pkg/block/local/adapter.go

```diff
@@ -152,21 +152,24 @@ func (l *Adapter) Path() string {
 	return l.path
 }
 
-func (l *Adapter) Put(_ context.Context, obj block.ObjectPointer, _ int64, reader io.Reader, _ block.PutOpts) error {
+func (l *Adapter) Put(_ context.Context, obj block.ObjectPointer, _ int64, reader io.Reader, _ block.PutOpts) (*block.PutResponse, error) {
 	p, err := l.extractParamsFromObj(obj)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	p = filepath.Clean(p)
 	f, err := l.maybeMkdir(p, os.Create)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	defer func() {
 		_ = f.Close()
 	}()
 	_, err = io.Copy(f, reader)
-	return err
+	if err != nil {
+		return nil, err
+	}
+	return &block.PutResponse{}, nil
```

Comment:
I think here it would make sense to take the mtime from the filesystem.  Unfortunately lakeFS on "local" NFS is a thing, and this code will run into time sync issues.

Reply:
I wanted to deal with the more urgent issue first and have fallbacks. 
I'll open a separate issue for that